### PR TITLE
Check if default is set to None

### PR DESCRIPTION
**Pull request by jnosal**

### CODE REVIEW

File: walrus/tests.py [hunk 1]
```diff
@@ -54,6 +54,12 @@ class Stat(BaseModel):
 class HashModel(BaseModel):
     data = HashField()
 
+class DefaultOption(BaseModel):
+    default_empty = JSONField()
+    default_dict = JSONField(default={})
```
Comment (coleifer) [inline walrus/tests.py]: You wouldn't want your default value to be mutable. This goes along with not using mutable objects as default function parameters.

Comment (coleifer) [inline walrus/tests.py]: So maybe remove the `default_dict` lines and this is great!

**Comment by jnosal** [comment]
removed according to suggestion ;)

**Comment by coleifer** [comment]
Odd, GitHub is saying there are conflicts and I cannot merge.

**Comment by jnosal** [comment]
Strange indeed, but it should be fine now :)

**Pull request closed without merging**